--- proj-4.4.5/src/Makefile.am~	Sun Jun  2 09:27:22 2002
+++ proj-4.4.5/src/Makefile.am	Sun Jun  2 09:27:22 2002
@@ -10,7 +10,7 @@
 cs2cs_SOURCES = cs2cs.c gen_cheb.c p_series.c
 nad2nad_SOURCES = nad2nad.c 
 nad2bin_SOURCES = nad2bin.c
-geod_SOURCES = geod.c geod_set.c geod_for.c geod_inv.c emess.c geodesic.h
+geod_SOURCES = geod.c geod_set.c geod_for.c geod_inv.c geodesic.h
 
 proj_LDADD = libproj.la
 cs2cs_LDADD = libproj.la
